ethereum/hardhat.config.ts

```diff
@@ -91,13 +113,13 @@ const config: HardhatUserConfig = {
           ? [process.env.ETHEREUM_GOERLI_PRIVATE_KEY]
           : [],
     },
-    "optimism-kovan": {
-      url: `https://opt-kovan.g.alchemy.com/v2/${
-        process.env.OPTIMISM_KOVAN_API_KEY ?? ""
+    "optimism-goerli": {
+      url: `https://opt-goerli.g.alchemy.com/v2/${
+        process.env.OPTIMISM_GOERLI_API_KEY ?? ""
       }`,
       accounts:
-        process.env.OPTIMISM_KOVAN_PRIVATE_KEY !== undefined
-          ? [process.env.OPTIMISM_KOVAN_PRIVATE_KEY]
+        process.env.OPTIMISM_GOERLI_PRIVATE_KEY !== undefined
+          ? [process.env.OPTIMISM_GOERLI_PRIVATE_KEY]
```

Comment:
Tableland doesn't support Optimism Kovan anymore, so I'm guessing those configs are needed anymore. Although now that I'm thinking about it, i guess we could still deploy Rigs contracts to Optimism Kovan. Anyway, let me know what you think.